ci-operator/step-registry/openshift/e2e/openstack/qe/multijob/openshift-e2e-openstack-qe-multijob-workflow.yaml

```diff
@@ -0,0 +1,15 @@
+workflow:
+  as: openshift-e2e-openstack-qe-multijob
+  steps:
+    pre:
+      - ref: openstack-qe-gather-build-number
+    test:
+      - ref: openstack-qe-gather-test-result-conformance
+      - ref: openstack-qe-gather-test-result-csi-cinder
+      - ref: openstack-qe-gather-test-result-csi-manila
+      - ref: openstack-qe-gather-test-result-ccpmso
+      - ref: openstack-qe-gather-test-result-openstack
+      - ref: openstack-qe-gather-must-gather
+      - ref: openstack-qe-gather-test-report
+  documentation: |-
```

Comment:
Is the CI downstream really running on dual-stack?
If that is true, should the documentation here be updated?
If it's not true, should the name of the jobs be updated to not mention dual-stack?